pkg/util/iptables/iptables.go

```diff
@@ -328,10 +328,11 @@ func (runner *runner) checkRuleWithoutCheck(table Table, chain Chain, args ...st
 
 	// Sadly, iptables has inconsistent quoting rules for comments.
 	// Just unquote any arg that is wrapped in quotes.
-	argsCopy := make([]string, len(args))
-	copy(argsCopy, args)
-	for i := range argsCopy {
-		unquote(&argsCopy[i])
+	var argsCopy []string
+	for i := range args {
+		tmpField := args[i]
+		unquote(&tmpField)
+		argsCopy = append(argsCopy, strings.Fields(tmpField)...)
```

Comment:
This is because, 
while the ```args``` supplied to checkRuleWithoutCheck function has multi-word comment strings as _ONE_ string, (eg: ```"handle ClusterIPs; NOTE: this must be before the NodePort rules"``` is 1 string),  
stringset from ```strings.Fields(`iptables-save -t nat`)``` splits at whitespace boundary, and has such multi-word comment as multiple strings. (eg: ```"handle ClusterIPs; NOTE: this must be before the NodePort rules"``` will be a set of 10 strings each consisting ["handle, ClusterIPs;, NOTE, ...]```)

So, the len(fields) would never equal len(args)+2 when there were multi-word comments, and this would result in duplicate rules added every few seconds.